projects/demo/src/modules/experimental/textfield/examples/1/index.html

```diff
@@ -0,0 +1,219 @@
+<div
+    class="new"
+    [tuiTextfieldCleaner]="true"
+>
+    <h2 class="tui-heading-5">Label inside</h2>
+    <tui-textfield
+        *ngFor="let size of sizes"
+        iconLeft="tuiIconSearchLarge"
+        iconRight="tuiIconSettingsLarge"
+        [filler]="filler"
+        [tuiTextfieldSize]="size"
+    >
+        <label tuiLabel>I am a label</label>
+        <input
+            placeholder="I am placeholder"
+            tuiTextfieldd
+            [(ngModel)]="value"
+        />
+        <tui-icon
+            icon="tuiIconBellLarge"
+            ngProjectAs="tuiSide"
```

Comment:
Is it necessary? I didn't find `select="tuiSide"`

Reply:
Old idea